class-gf-razorpay.php

```diff
@@ -0,0 +1,336 @@
+<?php
+
+require_once ('razorpay-sdk/Razorpay.php');
+use Razorpay\Api\Api;
+
+GFForms::include_payment_addon_framework();
+
+class GFRazorpay extends GFPaymentAddOn
+{
+    protected $_version                  = GF_RAZORPAY_VERSION;
+    protected $_min_gravityforms_version = '1.9.3';
+    protected $_slug                     = 'razorpay-gravity-forms';
+    protected $_path                     = 'razorpay-gravity-forms/razorpay.php';
+    protected $_full_path                = __FILE__;
+    protected $_url                      = 'http://www.gravityforms.com';
+    protected $_title                    = 'Gravity Forms Razorpay Add-On';
+    protected $_short_title              = 'Razorpay';
+    protected $_supports_callbacks       = true;
+
+    // Permissions
+    protected $_capabilities_settings_page = 'gravityforms_razorpay';
+    protected $_capabilities_form_settings = 'gravityforms_razorpay';
+    protected $_capabilities_uninstall     = 'gravityforms_razorpay_uninstall';
+
+    // Automatic upgrade enabled
+    protected $_enable_rg_autoupgrade = true;
+
+    private static $_instance = null;
+
+    const GF_RAZORPAY_KEY    = 'gf_razorpay_key';
+    const GF_RAZORPAY_SECRET = 'gf_razorpay_secret';
+    const RAZORPAY_ENTRY_ID  = 'gf_razorpay_entry_id';
+    const RAZORPAY_ORDER_ID  = 'razorpay_order_id';
+
+    //cookie set for one day
+    const COOKIE_DURATION    = 86400;
+
+    public static function get_instance()
+    {
+        if (self::$_instance === null)
+        {
+            self::$_instance = new GFRazorpay();
+        }
+
+        return self::$_instance;
```

Comment:
@riagarg5 Please refer to this: http://stackoverflow.com/a/694847/3160368

Kindly change it to the ternary operator. We use this regularly in our api codebase as well.